src/test/java/com/ui_init_setup/practiceproject/tests/ParaBankAccountTest.java

```diff
@@ -0,0 +1,44 @@
+package com.ui_init_setup.practiceproject.tests;
+
+import com.ui_init_setup.practiceproject.junit.extension.WebDriverSetup;
+import com.ui_init_setup.practiceproject.models.Account;
+import com.ui_init_setup.practiceproject.pages.ParaBankAccountOverviewPage;
+import com.ui_init_setup.practiceproject.pages.ParaBankLoginComponent;
+import com.ui_init_setup.practiceproject.utils.APIUtil;
+import com.ui_init_setup.practiceproject.utils.Customers;
+import com.ui_init_setup.practiceproject.webDriver.ThreadLocalDriver;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith({WebDriverSetup.class})
+class ParaBankAccountTest {
+
+    private final String username = "john";
+    private final String password = "demo";
+
+    @Test
+    void fetchingUserAccountsList() {
```

Comment:
could you kindly explain the meaning of this test? maybe this can lead to a more descriptive name